web/modules/custom/job_board/src/Plugin/Block/RecruiterMenu.php

```diff
@@ -96,8 +96,8 @@ public function build() {
     );
     $cache->addCacheableDependency($user);
     if ($user->organization->isEmpty()) {
-      $cache->applyTo($build);
-      return $build;
+      //$cache->applyTo($build);
+      //return $build;
```

Comment:
We need to not commit these two lines.

Reply:
Of course, my bad! I've removed the forward slashes from both lines now and will push through.